cli/src/main/java/com/devonfw/tools/ide/tool/ide/IdeToolCommandlet.java

```diff
@@ -77,6 +85,14 @@ protected void configureWorkspace() {
     }
     try (Step step = this.context.newStep("Configuring workspace " + ideWorkspacePath.getFileName() + " for IDE " + this.tool)) {
       int errors = this.context.getWorkspaceMerger().merge(setupFolder, updateFolder, this.context.getVariables(), ideWorkspacePath);
+      try {
+        Files.copy(editorConfigPath, ideWorkspacePath.resolve(".editorconfig"),
+            StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Should the editorconfig only be overwritten or should it be merged instead?
If it has to be merged, then we need to use our merger for it.

Reply:
See #465 and #735 - so my expectation is also different on the long run. But to get this feature earlier, we start with this hardcoded quickfix.